### ISSUE TITLE
Telegram Bot "list index out of range"

### ISSUE THREAD

**Issue by Ednn0nd1au** [AmmeySaini/Chegg-Answer-Bot#22]
Hi,

I just tried and the telegram bot showed "list index out of range".

From terminal side nothing happend:
<img width="556" alt="image" src="https://user-images.githubusercontent.com/36107555/169660216-1c57e401-56fa-44d4-b701-089757bdb8d8.png">

**Comment by Ednn0nd1au** [comment]
Hi, I notice that this issue will happen on Linux and Windows platforms. Would you please kindly suggest any solutions on this~  @AmmeySaini

**Comment by Ednn0nd1au** [comment]
Hi, I notice that this issue will happen on Linux and Windows platforms. Would you please kindly suggest any solutions on this~  @AmmeySaini

**Comment by haribhuju** [comment]
was that when you are try to do setup.py

**Comment by avtar312** [comment]
> was that when you are try to do setup.py

Can you help me at telegram @Chigigv

**Comment by avtar312** [comment]
> was that when you are try to do setup.py

 File "C:\Users\nitin\Desktop\Chegg-Answer-Bot\helper.py", line 45, in try2FA
    f_id = factors[idd - 1]['id']
IndexError: list index out of range
PS C:\Users\nitin\Desktop\Chegg-Answer-Bot>

**Comment by haribhuju** [comment]
yeah this is because of the refresh token. it expires within 24 hrs if it is active otherwise if not active it expires within 30 mis.

**Comment by valkin123** [comment]
i'm also receiving this message even though I reset my tokens, any fix?

**Comment by aditiarora1** [comment]
Can anyone help me in installing this bot?
